### PR TITLE
Add force option to the command `container create`

### PR DESCRIPTION
**Pull request by acid-ant**
Validate policy before container creation

Signed-off-by: Anton Nikiforov <an.nikiforov@yadro.com>

### CODE REVIEW

File: cmd/neofs-cli/modules/container/create.go [hunk 1]
```diff
@@ -138,6 +152,8 @@ func initContainerCreateCmd() {
 	flags.StringVar(&containerName, "name", "", "container name attribute")
 	flags.BoolVar(&containerNoTimestamp, "disable-timestamp", false, "disable timestamp container attribute")
 	flags.StringVar(&containerSubnet, "subnet", "", "string representation of container subnetwork")
+	flags.BoolVarP(&force, commonflags.ForceFlag, commonflags.ForceFlagShorthand, false,
+		"force create container with not well-known policy")
```
Comment (fyrchik) [inline cmd/neofs-cli/modules/container/create.go:156]: What about `skip placement validity check`?

**Review by fyrchik — DISMISSED** [review]

File: cmd/neofs-cli/modules/container/create.go [hunk 2]
```diff
@@ -39,6 +40,23 @@ It will be stored in sidechain when inner ring will accepts it.`,
...
+		cli := internalclient.GetSDKClientByFlag(cmd, key, commonflags.RPC)
+
+		if !force {
+			var prm internalclient.NetMapSnapshotPrm
+			prm.SetClient(cli)
+			resmap, err := internalclient.NetMapSnapshot(prm)
+			if err != nil {
+				common.ExitOnErr(cmd, "unable to get netmap snapshot: %w", err)
+			}
+			var nm = resmap.NetMap()
```
Comment (cthulhu-rider) [inline cmd/neofs-cli/modules/container/create.go:53]: Maybe directly call `resmap.NetMap().ContainerNodes(...)` below?

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,6 +11,7 @@ Changelog for NeoFS Node
 - Validate storage node configuration before node startup (#1805)
 - `neofs-node -check` command to check the configuration file (#1805)
 - `flush-cache` control service command to flush write-cache (#1806)
+- Validate policy before container creation, add `--force` option to skip validation (#1704)
```
Comment (cthulhu-rider) [inline CHANGELOG.md:14]: I'd place `add --force` statement to `Updating from...` section, what do u think?

File: cmd/neofs-cli/modules/container/create.go [hunk 4]
```diff
@@ -39,6 +40,23 @@ It will be stored in sidechain when inner ring will accepts it.`,
...
+			_, err = nm.ContainerNodes(*placementPolicy, nil)
+			if err != nil {
+				common.ExitOnErr(cmd, "could not build container nodes based on given placement policy: %w", err)
```
Comment (carpawell) [inline cmd/neofs-cli/modules/container/create.go:56]: do we need any comment about the `--force` flag in the error msg here? to make it easier for a user to find out about the new flag?

File: cmd/neofs-cli/modules/container/create.go (hunk 1 above)
Comment (acid-ant) [reply cmd/neofs-cli/modules/container/create.go]: Sounds better, updated.

File: cmd/neofs-cli/modules/container/create.go (hunk 2 above)
Comment (acid-ant) [reply cmd/neofs-cli/modules/container/create.go]: Thanks, addressed

File: CHANGELOG.md (hunk 3 above)
Comment (acid-ant) [reply CHANGELOG.md]: Done

File: cmd/neofs-cli/modules/container/create.go [hunk 5]
```diff
@@ -39,6 +40,23 @@ It will be stored in sidechain when inner ring will accepts it.`,
...
 		common.ExitOnErr(cmd, "", err)
 
+		key := key.GetOrGenerate(cmd)
+		cli := internalclient.GetSDKClientByFlag(cmd, key, commonflags.RPC)
+
+		if !force {
+			var prm internalclient.NetMapSnapshotPrm
+			prm.SetClient(cli)
+			resmap, err := internalclient.NetMapSnapshot(prm)
+			if err != nil {
```
[the remark this answers is not in the document]
Comment (acid-ant) [reply cmd/neofs-cli/modules/container/create.go]: Sorry, fixed.

File: cmd/neofs-cli/modules/container/create.go (hunk 4 above)
Comment (acid-ant) [reply cmd/neofs-cli/modules/container/create.go]: Added more info in error message.

**Review by cthulhu-rider — APPROVED** [review]

**Review by fyrchik — APPROVED** [review]

**Comment by fyrchik** [comment]
@acid-ant there are conficts with master

**Review by cthulhu-rider — APPROVED** [review]

**Review by carpawell — APPROVED** [review]

**Pull request merged**